### ISSUE TITLE
Proxy command not working after update

### ISSUE THREAD

**Issue by jgnieuwhof** [superfly/flyctl#975]
**Describe the bug**

My team has a github action that began failing around 3pm ET yesterday, with the following error:
```console
> flyctl proxy 5432 -a <my-postgres-app>
Oops, something went wrong! Could you try that again?
```

The pipeline uses the latest version of `flyctl`, which received a proxy related [release](https://github.com/superfly/flyctl/releases/tag/v0.0.326) around the same time.

Pinning `flyctl` to version `0.0.325` has restored the previous (working) behaviour.

**System Information**
* OS: `ubuntu-20.04`
* Fly: `>= 0.0.326`

**More information**
* I've upgraded fly locally and reproduced the behaviour there as well.
* Tried specifying the new `--org` option to see if that would solve it (it didn't)
* Tried the new `--select` option, and received this error message:
```console
> flyctl proxy 5432 --app <my-postgres-app> --org <my-org> --select
Error --app required when --select flag provided
```

**Comment by jsierles** [comment]
Sorry about that - this should be fixed in #976 and released shortly.

**Comment by jgnieuwhof** [comment]
All good, thanks for the quick follow up!
